internal/graph/check.go

```diff
@@ -214,12 +214,9 @@ func (cc *ConcurrentChecker) checkDirect(ctx context.Context, crc currentRequest
 	hadDirectResult := false
 	hadDispatchedResult := false
 	defer (func() {
-		estimatedQueryCount := 0.0
-		if hadDirectResult {
-			estimatedQueryCount++
-		}
-		if hadDispatchedResult {
-			estimatedQueryCount++
+		estimatedQueryCount := 1.0
+		if !hadDirectResult && hadDispatchedResult {
+			estimatedQueryCount = 2
 		}
 		estimatedDirectDispatchQueryHistogram.Observe(estimatedQueryCount)
 	})()
```

Comment:
do we need the wrapping parentheses?

Reply:
Maybe not, but I always prefer it when using function definitions under defers